### ISSUE TITLE
The rpm Python binding may be missing a license 

### ISSUE THREAD

**Issue by pombredanne** [rpm-software-management/rpm#1236]
What would be the license for this? LGPL like librpm or GPL like the rest of the rpm code?
(hint LGPL would be a great answer to integrate in Apache-licensed Python scripts)
I could not find any hint of a license that may specifically  apply to this
And it is not in the setup.py https://github.com/rpm-software-management/rpm/blob/c4f7f871e0d0b7faaaa16e93736b8dd39db200e2/python/setup.py.in#L44

@spotrh @Conan-Kudo would you know by chance?

I maintain a pure-python RPM headers parser at https://github.com/nexB/scancode-toolkit/blob/develop/src/packagedcode/pyrpm.py but I would much prefer using the standard.

**Comment by pmatilai** [comment]
The python bindings are part of the rpm source tree, hence COPYING applies:

> The entire code base may be distributed under the terms of the GNU General
Public License (GPL), which appears immediately below.  Alternatively,
all of the source code in the lib and rpmio subdirectories of the RPM source
code distribution as well as any code derived from that code may instead be
distributed under the GNU Library General Public License (LGPL), at the
choice of the distributor.

In other words, GPLv2.

The license should be specified in the Python distutils module description though, so thanks for reporting.

**Comment by pombredanne** [comment]
@pmatilai you wrote:

> In other words, GPLv2.

That's what I thought (with some hope it would be LGPL like for librpm)

> The license should be specified in the Python distutils module description though, so thanks for reporting.

That would be a nice improvement for clarity! Thank you!

**Comment by pombredanne** [comment]
@pmatilai thank you ++
